testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/LoggedInPageHeader.java

```diff
@@ -27,40 +27,24 @@
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
 public class LoggedInPageHeader extends AbstractHeader {
-    @FindBy(xpath = "//*[@data-testid='options']//a[text() = 'Sign out']")
+    @FindBy(xpath = "//div[@id='app']//header[@class='pf-v5-c-page__header']//span[text() = 'Sign out']")
     private WebElement logoutBtn;
-    @FindBy(xpath = "//*[@data-testid='options-kebab']//a[text() = 'Sign out']")
-    private WebElement logoutBtnMobile;
 
-    @FindBy(id = "referrerLink")
-    private WebElement referrerLink;
-    @FindBy(id = "referrerMobileLink")
-    private WebElement referrerLinkMobile;
-
-    @FindBy(xpath = "//*[@data-testid='options']")
+    @FindBy(xpath = "//div[@id='app']//header[@class='pf-v5-c-page__header']//button[@class='pf-v5-c-menu-toggle']")
```

Comment:
Same here, let's try to stick to classes only.
```suggestion
    @FindBy(xpath = "//[@class='pf-v5-c-page__header']//[@class='pf-v5-c-menu-toggle']")
```